services/DemographicsService.js

```diff
@@ -6,35 +6,74 @@
 class DemographicsService {
 
     /**
-     * Constructs a new service backed by the provided
-     * `dbSession` database session
-     * @param dbSession a connected database session
+     * Constructs a new service backed by the AWS docClient
      */
     constructor(dbSession) {
-        this.session = dbSession;
+        var AWS = require("aws-sdk");
+
+        AWS.config.update({
+          region: "us-east-1",
+          endpoint: "https://dynamodb.us-east-1.amazonaws.com"
+        });
+        //AWS.config.setPromisesDependency(null);
```

Comment:
@garzillo we should probaby use the `Promise`-based AWS API and rather than wrapping the old one as you've done below.